packages/crayons-extended/platform-ui/src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+    <title>Stencil Component Starter</title>
+
+    <script type="module" src="/build/platform-ui.esm.js"></script>
+    <script nomodule src="/build/platform-ui.js"></script>
+  </head>
+  <body>
+    <my-component first="Stencil" last="'Don't call me a framework' JS"></my-component>
```

Comment:
please remove this `my-component` . Maybe add some example of a component present in platformui package